src/main.c

```diff
@@ -60,6 +60,7 @@ void app_main() {
                        cmd.p1,
                        cmd.p2,
                        cmd.lc,
+                       cmd.lc,
```

Comment:
This was removed on another PR, but it was making tests fail.
Fortunately we caught it here.
The issue is that `CData=%.*H` requires 2 arguments, the length and data to print in hex, and lc is the data length, so we need 2 `cmd.lc` for this printf statement.